packages/quantic/force-app/main/default/lwc/quanticUtils/quanticUtils.js

```diff
@@ -89,4 +89,14 @@ export const keys = {
   ENTER: 'Enter',
   ARROWUP: 'ArrowUp',
   ARROWDOWN: 'ArrowDown',
-};
\ No newline at end of file
+};
+
+export function getItemfromLocalStorage(key) {
```

Comment:
I'd suggest to rename `getItemfromLocalStorage` to  `getItemFromLocalStorage`. It is a little easier to read.